packages/components/ng-ovh-sso-auth/package.json

```diff
@@ -43,6 +43,6 @@
   "peerDependencies": {
     "angular": "^1.5.0",
     "angular-cookies": "^1.5.0",
-    "jquery": "^2.1.3"
+    "whatwg-fetch": "^3.0.0"
```

Comment:
This is not an angular dependency, would it make sense to add it as `dependency` instead ?

Reply:
oh yeah